### PR TITLE
Added extra request to registry/{package}/latest for .readmeText attribute

### PR DESCRIPTION
**Pull request by tsafin**
We have discovered the harder way that /-/all request to registry does not return .readmeText attribute,
but particular request for /:packageName/:version does return all attributes, so invoking subrequest for this attribute.

### CODE REVIEW

File: src/services/dataSrvc.js [hunk 1]
```diff
@@ -38,5 +38,16 @@ angular.module('app').factory('DataSrvc', ['$http', '$q', '$location', function
           })
 
       },
+      getUrlData: function (url, cb) {
```
Comment (daimor) [inline src/services/dataSrvc.js:41]: Should be changed to method getPackageData, which returns data by package name, and repository URL as a constant in this class.

File: src/controllers/repoCtrl.js [hunk 2]
```diff
@@ -3,9 +3,13 @@ angular.module('app').controller('RepoCtrl', ['$scope', 'DataSrvc', '$routeParam
   $scope.readmeText = '';
 
   DataSrvc.getData(function (data) {
-    $scope.data = data[$routeParams.name];
-    $scope.readmeText = $scope.data.readmeText;
-    console.log($scope.readmeText);
+    var packageName = $routeParams.name;
+    $scope.data = data[packageName];
+
+    DataSrvc.getUrlData('https://cpmisc.smileupps.com/' + packageName + '/latest', function(data) {
```
Comment (daimor) [inline src/controllers/repoCtrl.js]: conditional loading readmeText, if it was not loaded before.

**Review by daimor — CHANGES_REQUESTED** [review]

File: src/services/dataSrvc.js (hunk 1 above)
Comment (tsafin) [reply src/services/dataSrvc.js:41]: [Actually it is generic enough that's why I named it getUrlData.] 
In this case it's actually not a _constant_ but rather a function of the url used in the getData (we should name it getAllData). The hardcoded url is a bad hack to see it working. I'll update it to use not-constant value in the next step.

File: src/controllers/repoCtrl.js (hunk 2 above)
Comment (tsafin) [reply src/controllers/repoCtrl.js]: Yeah, good point. Will change

**Review by daimor — APPROVED** [review]

**Pull request merged**